scylla-cql/src/errors.rs

```diff
@@ -348,6 +348,10 @@ pub enum BadQuery {
     #[error("Passed invalid keyspace name to use: {0}")]
     BadKeyspaceName(#[from] BadKeyspaceName),
 
+    /// Too many queries in the batch statement
+    #[error("Number of Queries in Batch Statement has exceeded the max value of 65,536")]
+    TooManyQueriesInBatchStatement,
```

Comment:
The maximum value of `u16` is 65,535 - one less than 65,536.
https://doc.rust-lang.org/std/primitive.u16.html#associatedconstant.MAX

Reply:
It'd also be nice to display the number of queries that the user has passed. This can be done by adding `(usize)` to the enum variant and displaying it with `{0}`.

Reply:
Nice Catch. Fixed